src/internal/storage/chunk/client.go

```diff
@@ -87,13 +87,13 @@ func (c *trackedClient) beforeUpload(ctx context.Context, chunkID ID, md Metadat
 		pointsTo = append(pointsTo, cid.TrackerID())
 	}
 	chunkTID := chunkID.TrackerID()
-	if err := dbutil.WithTx(ctx, c.db, func(tx *pachsql.Tx) (retErr error) {
+	if err := dbutil.WithTx(ctx, c.db, func(cbCtx context.Context, tx *pachsql.Tx) (retErr error) {
```

Comment:
Calling this `cbCtx` means that any transaction code that was using a context, is still probably using that context.  Most of that code will be referring to a context `ctx`, which we would shadow by calling this `ctx`.

I can see the reason for doing this to keep the change non-functional, but we should renaming these back to `ctx`, and ensuring that it is the only context used in the transaction.